monitorcontrol/monitorcontrol.py

```diff
@@ -135,6 +159,160 @@ def _get_vcp_feature(self, code: vcp.VCPCode) -> int:
         current, maximum = self.vcp.get_vcp_feature(code.value)
         return current
 
+    def _extract_a_cap(self, caps_str: str, key: str) -> str:
```

Comment:
This does not use anything from `self`, would be good to move outside of the class, makes unit testing easier.